src/components/media-player/media-range-selector.yml

```diff
@@ -4,9 +4,21 @@ modalTitle:
 playbackRange:
   en: "Playback range: from {from, select, start {the beginning} other {timecode {from}}} to {to, select, end {the end} other {timecode {to}}}"
   de: "Abspielbereich: von {from, select, start {Anfang} other {Time-Code {from}}} bis {to, select, end {Ende} other {Time-Code {to}}}"
+selectRangeLabel:
+  en: Set current playback timecode
+  de: Setz den aktuellen Wiedergabe Time-Code
```

Comment:
"Setz**e**" and "Wiedergab**e-T**ime-Code"

Maybe an even less technical alternative: "Set current playback position" / "Setze die aktuelle Wiedergabeposition" ???

Reply:
Sounds good, I will change both texts 👍

Reply:
Done!